libs/block-content/website/src/lib/image/image-block.tsx

```diff
@@ -12,18 +13,27 @@ export const isImageBlock = (block: Block): block is ImageBlockType =>
   block.__typename === 'ImageBlock'
 
 export const ImageBlockWrapper = styled('figure')`
+  display: grid;
+  gap: ${({theme}) => theme.spacing(2)};
   margin: 0;
   max-width: 100%;
 `
 
+export const ImageBlockCaption = styled('figcaption')``
+
+const imageStyles = css`
+  object-fit: cover;
+  aspect-ratio: 1.85611511;
```

Comment:
do we want this many decimal places? wouldn't maximum of two or three be enough?

Reply:
Just c&p from rons comments in the design